app/models/ingest_job.rb

```diff
@@ -999,7 +1010,10 @@ def anndata_summary_props
       studyAccession: study.accession,
       trigger:,
       jobStatus: job_status,
-      numFilesExtracted: num_files_extracted
+      numFilesExtracted: num_files_extracted,
+      machineType: params_object.machine_type,
+      error: error_action,
```

Comment:
AIUI, this is equivalent to the `action` property in `ingest` events, which we use in non-AnnData contexts.

Logging this data via the same `action` property in AnnData contexts -- and logging it for failure (like here) _as well as_ success -- seems like it would ease analysis.  E.g. it would allow us to use the same property in Mixpanel queries about ingest, whether the ingest was AnnData or not.  That could help e.g. compare error causes for the same underlying classic file type (e.g. metadata, cluster) whether derived from AnnData or not.  It could also help us sum up ingests by classic file type, whether the parent ingest was AnnData or not.

(Intuitively, I'd expect `error` to log a message or a code, not the ingest action / job run type.)

Am I overlooking something?  The suggestion below is part of a batch.

```suggestion
      action: ,
```